kinetic/distribution.yaml

```
@@ -4183,15 +4183,17 @@ repositories:
       version: kinetic-devel
     release:
       packages:
+      - heron_control
       - heron_description
       - heron_msgs
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/clearpath-gbp/heron-release.git
-      version: 0.3.0-0
+      version: 0.3.1-0
     source:
       type: git
       url: https://github.com/heron/heron.git
+    status: maintained
```

I'm not sure how this was passing before, but we need a branch name version tag in the source entry.

```suggestion
      version: kinetic-devel
    status: maintained
```